source/includes/code-examples/DeleteOneSync.cs

```diff
@@ -0,0 +1,70 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class DeleteOneSync
```

Comment:
I would suggest adding a Restore() method to both the sync and async classes, that adds the deleted file back into the collection at the end of the program run. This will make sure that the users sample data doesn't permanently change and makes the apps re-runnable.